CHANGELOG.md

```diff
@@ -836,9 +835,40 @@ Non-backwards compatible changes
   Therefore, only _declarations_ of `PartialOrder` records will need their field names
   updating.
 
-* NB (issues #1214 #2098) the corresponding situation regarding the `flip`ped
-  relation symbols `_≥_`, `_>_` (and their negated versions!) has not (yet)
-  been addressed.
+* NB (issue #2098) the corresponding situation regarding the `flip`ped
+  relation symbols `_≥_`, `_>_` has not (yet) been addressed.
+
+### (Issue #1214) Reorganisation of the introduction of negated relation symbols under `Relation.Binary`
+
+* Previously, negated relation symbols `_≰_` (for `Poset`) and `_≮_` (`TotalOrder`)
+  were introduced in the corresponding `Relation.Binary.Properties` modules, for re-export.
+
+* Now they are introduced as definitions in the corresponding `Relation.Binary.Bundles`,
+  together with, for uniformity's sake, an additional negated symbol `_⋦_` for `Preorder`.
+  NB `_≮_` is now introduced *earlier* in the hierarchy, at `StrictPartialOrder`, which
+  incurs some rather complicated deprecation gymnastics...
+
+* Accordingly, in fact we now have `_≰_` (in `Poset`) as a renamed `public` re-export
+  from `_⋦_` in `Preorder`, with `TotalPreorder` inheriting `_⋦_` from `Preorder`.
+
+* As knock-on changes, public re-exports of `Preorder`, `Poset`, `StrictPartialOrder`
+  in derived bundles also now need to re-export the various new symbols accordingly.
+
+* Backwards compatibility has been maintained, with deprecated definitions in the
+  corresponding `Relation.Binary.Properties` modules, and the corresponding client
+  client module `import`s being adjusted accordingly.
+
+* Elsewhere under `Relation.Binary.Properties` etc. the use of *explicitly* negated
+  relation symbols have now been replaced by their definitionally equal counterparts
+  using the new symbols accordingly.
+
+* NB modules such as `Relation.Binary.Construct.NonStrictToStrict` which operate
+  only on the the underlying `Structures` such as `IsPartialOrder` etc., are unable
+  to make use of the new symbols.
```

Comment:
Again, I don't think this is relevant. The API of these modules haven't changed in anyway.

Reply:
OK